### PR TITLE
message-card-tools-restyling

### PR DESCRIPTION
**Pull request by VitalyMashanov**

### CODE REVIEW

File: src/components/message/message-card/MessageCardTools.tsx [hunk 1]
```diff
@@ -64,10 +64,10 @@ const MessageCardTools = ({
 
 	const viewTypes = message.body
 		? [
-				MessageViewType.JSON,
 				MessageViewType.FORMATTED,
```
Comment (rzvdv) [inline src/components/message/message-card/MessageCardTools.tsx:67]: why has order changed? i think json and formatted view types should be next to each other

File: src/components/message/message-card/MessageCardTools.tsx [hunk 2]
```diff
@@ -172,22 +160,16 @@ const MessageCardTools = ({
 					(messageViewType === MessageViewType.JSON ||
 						messageViewType === MessageViewType.FORMATTED) && (
 						<div className='message-card-tools__controls-group'>
-							{['body', 'fields'].map(copyOption => (
+							{['fields'].map(copyOption => (
```
Comment (rzvdv) [inline src/components/message/message-card/MessageCardTools.tsx:163]: we don't need body copying anymore? if that's so array with 1 item looks weird we should refactor that

**Review by rzvdv — CHANGES_REQUESTED** [review]
Pick any message that fits in one line, switch to ASCII mode, option window now has scrollbar

File: src/components/message/message-card/MessageCardTools.tsx (hunk 1 above)
Comment (VitalyMashanov) [reply src/components/message/message-card/MessageCardTools.tsx:67]: according to https://www.figma.com/file/vB9EyLv1GBE3vJqy2Hxk08/TH2-New-Report-Style-Updates?node-id=2738%3A85943

Comment (rzvdv) [reply src/components/message/message-card/MessageCardTools.tsx:67]: i think previous order makes more sense. let's leave it as it was

**Review by rzvdv — CHANGES_REQUESTED** [review]
Either we a make temporary solution for copying message without deleting part of functionality or we wait until sketch is updated, otherwise i don't think it should be merged

**Pull request closed without merging**